src/vs/workbench/contrib/bulkEdit/browser/preview/bulkEdit.contribution.ts

```diff
@@ -122,7 +122,7 @@ class BulkEditPreviewContribution {
 			const choice = await this._dialogService.show(
 				Severity.Info,
 				localize('overlap', "Another refactoring is being previewed."),
-				[localize('cancel', "Cancel"), localize('continue', "Continue")],
+				[localize('continue', "Continue"), localize('cancel', "Cancel")],
```

Comment:
doesn't this break checking of `choice` further down

Reply:
via https://github.com/microsoft/vscode/pull/155049/commits/517600ee1f99f27abd202e62d69ecc600db20c68